tictactoe/src/main/java/com/github/martingaston/tictactoe/api/Encode.java

```diff
@@ -4,10 +4,10 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 class Encode {
-    static String from(GameJSON gameJson) throws JsonProcessingException {
+    static String from(JsonOutgoing jsonOutgoing) throws JsonProcessingException {
         var mapper = new ObjectMapper();
         mapper.configure(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY, true);
-        return mapper.writeValueAsString(gameJson);
+        return mapper.writeValueAsString(jsonOutgoing);
```

Comment:
I feel like that the two classes could be one:

class GameJson {
   public Game objectify(Json game) {}
   pubic Json parse(Game game) {}
}